integration/README.md

```diff
@@ -6,7 +6,12 @@
 
 ## Prereqs
 
-Install docker-compose
+Install docker-compose <https://docs.docker.com/compose/install/>
+
+```bash
+ $ docker-compose -v
+docker-compose version 1.29.2, build unknown
```

Comment:
[1.29.2 is over a year old](https://github.com/docker/compose/releases/tag/1.29.2).. Can you also note this supported version which I am running:

```
vagrant@bullseye:/git/opi-poc/integration/opi-api$ docker compose version
Docker Compose version v2.6.0
vagrant@bullseye:/git/opi-poc/integration/opi-api$
```

[v2.6.0](https://github.com/docker/compose/releases/tag/v2.6.0)